src/components/AddItemForm.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { withFirestore } from 'react-firestore';
+import '../CSS/AddItemForm.css';
+class AddItemForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      next_purchase: null,
+      last_purchase: null,
+      user_token: 'shpongle',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSchedule = this.handleSchedule.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ name: event.target.value });
+  }
+
+  handleSchedule(event) {
+    this.setState({ next_purchase: event.target.value });
```

Comment:
We will expect to use this value as a number later on to calculate the next purchase date. What data type is `event.target.value` here? Is it a number? If not, how could we make it one?